### PR TITLE
ISSUE-294 Fixed broken thumbnail label

### PR DESCRIPTION
**Pull request by vyushin**
Pull request by issue #294

New label view:
image

### CODE REVIEW

File: styles/scss/image-gallery-no-icon.scss [hunk 1]
```diff
@@ -287,6 +287,10 @@ $ig-transparent: rgba(0, 0, 0, 0) !default;
     margin-left: 2px;
   }
 
+  & > div {
```
Comment (xiaolin) [inline styles/scss/image-gallery-no-icon.scss]: Probably worth adding a className to the `div`. Maybe call it `image-gallery-thumb-inner`

File: src/ImageGallery.jsx [hunk 2]
```diff
@@ -913,9 +913,11 @@ export default class ImageGallery extends React.Component {
...
           onError={onThumbnailError}
         />
-        <div className='image-gallery-thumbnail-label'>
-          {item.thumbnailLabel}
-        </div>
+        {item.thumbnailLabel && (
+          <div className='image-gallery-thumbnail-label'>
+            {item.thumbnailLabel}
+          </div>
+        )}
```
Comment (xiaolin) [inline src/ImageGallery.jsx]: Nice catch, can we adjust the style to look like below without the extra `()`
```
{
  item.thumbnailLabel &&
    <div className='image-gallery-thumbnail-label'>
      {item.thumbnailLabel}
    </div>
}
```

**Comment by vyushin** [comment]
@xiaolin, Thanks for code review! I have fixed it.

File: src/ImageGallery.jsx (hunk 2 above)
Comment (vyushin) [reply src/ImageGallery.jsx]: Done

File: styles/scss/image-gallery-no-icon.scss (hunk 1 above)
Comment (vyushin) [reply styles/scss/image-gallery-no-icon.scss]: Done

**Pull request merged**